docs/dataservice-read-catalog-example.md

```diff
@@ -9,7 +9,7 @@ const std::string gKeyId(""); // your here.access.key.id
 const std::string gKeySecret(""); // your here.access.key.secret
```

Comment:
Please mark for next commit that we need to align these with the coding style.
https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
I changed the naming of global and local variables according to the C++ coding style. Please, check both examples.cpp